### ISSUE TITLE
rm client is not connected

### ISSUE THREAD

**Issue by guixincui** [seata/seata#680]
- [x] I have searched the [issues](https://github.com/alibaba/fescar/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
undo log can not rollback
when run 'fescar-example' project , fescar-server generate a file 'root.data', and mysql undo_log generate a rollback_info.
and then restart the fescar-server, it throws that exception

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
2019-03-28 19:27:50.845 INFO [RetryRollbacking_1]com.alibaba.fescar.server.coordinator.DefaultCore.doGlobalRollback:282 -Exception rollbacking branch BR:2007485733/2007485730
java.lang.RuntimeException: rm client is not connected. dbkey:jdbc:mysql://172.16.70.20:3306/fescar_order,clientId:dubbo-demo-order-service:172.16.3.200:56684
	at com.alibaba.fescar.core.rpc.netty.RpcServer.sendSyncRequest(RpcServer.java:217)
	at com.alibaba.fescar.core.rpc.netty.RpcServer.sendSyncRequest(RpcServer.java:236)
	at com.alibaba.fescar.server.coordinator.DefaultCoordinator.branchRollback(DefaultCoordinator.java:180)
	at com.alibaba.fescar.server.coordinator.DefaultCore.doGlobalRollback(DefaultCore.java:260)
	at com.alibaba.fescar.server.coordinator.DefaultCoordinator.handleRetryRollbacking(DefaultCoordinator.java:224)
	at com.alibaba.fescar.server.coordinator.DefaultCoordinator.access$000(DefaultCoordinator.java:70)
	at com.alibaba.fescar.server.coordinator.DefaultCoordinator$1.run(DefaultCoordinator.java:278)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset$$$capture(FutureTask.java:308)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
```

### Ⅲ. Describe what you expected to happen

It should roll back normally when restart fescar-server

### Ⅵ. Environment:

- JDK version : 1.8
- OS : MacOS
- Others:

**Comment by xingfudeshi** [comment]
i will check.

**Comment by slievrly** [comment]
it already fix in 0.4.1

**Comment by xingfudeshi** [comment]
Will close this issue as the related problem has been fixed.if you have further question about this issue please comment here.thanks.
